parachains/runtimes/assets/statemint/src/lib.rs

```diff
@@ -800,6 +804,9 @@ impl_runtime_apis! {
 			use frame_system_benchmarking::Pallet as SystemBench;
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
 
+			type XcmBalances = pallet_xcm_benchmarks::fungible::Pallet::<Runtime>;
+			type XcmGeneric = pallet_xcm_benchmarks::generic::Pallet::<Runtime>;
+
 			let mut list = Vec::<BenchmarkList>::new();
 			list_benchmarks!(list, extra);
```

Comment:
something's up with this spacing. Also do type XcmBalances and XcmGeneric need to be defined here? I can't see how they would be referenced here, and they're defined below. Maybe I am missing something.

Reply:
Yeah, I'm going to check my CLion config. 
As far as the type definitions are concerned - I just used a similar setup from polkadot to get this going, definitely need to see why this is defined twice and eliminate the extra declaration if possible.

Reply:
@gilescope https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/1700740 for some context on why this has to be defined twice. 
1. https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/1700740
2. https://gitlab.parity.io/parity/mirrors/cumulus/builds/1700893

Reply:
Yes this is needed sometimes as alias for the benchmarks. Usually this is done with a `use` statement to reduce the confusion. tWO things:  
- Please add a comment why this is needed, so future devs will be less confused about why this is needed.
- Try out if A `use` statement works, that would integrate better with the other code.

Reply:
@ggwpez I'm trying out defining those types globally to see if that solves the issue. Thanks!